xml/System.Text.Json/JsonElement.xml

```diff
@@ -777,9 +789,8 @@ This method does not parse the contents of a JSON string value.
         <summary>Gets the value at a specified index when the current value is an <see cref="F:System.Text.Json.JsonValueKind.Array" />.</summary>
```

Comment:
Looks like we are missing the docs from source for the the other property, ValueKind:
https://github.com/dotnet/dotnet-api-docs/blob/c7324dff76601c7fdfeb798358fe63a43e61db3a/xml/System.Text.Json/JsonElement.xml#L1527-L1547

https://github.com/dotnet/corefx/blob/9e994a5923d426f0e64ed3b8f9b93df6f214a42a/src/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs#L32-L38